README.md

```diff
@@ -1 +1,180 @@
-# budgetapp
\ No newline at end of file
+<a name="readme-top"></a>
+
+- [💰 budgetapp ](#budgetapp-)
+  - [🚀 Live Demo  ](#-live-demo--)
+    - [Live Demo](#live-demo)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Installation ](#installation-)
+    - [database ](#database-)
+    - [Usage ](#usage-)
+    - [Run tests ](#run-tests-)
+  - [✍️ Authors ](#️-authors-)
+    - [👤 **Patrick Mukula**:](#-Patrick Mukula)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [💖 Show your support ](#-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝  License ](#--license-)
+
+
+# 💰 budgetapp <a name="about-project"></a>
```

Comment:
- Lastly, kindly update the title of your PR to make it more descriptive and in line with your project.